code/run_project.do

```diff
@@ -0,0 +1,39 @@
+clear all
+clear mata
+set more off
+
+**********************IMPORTANT**************************
+*Change the paths to match path on your computer locally*
+*********************************************************
+//Path of repository
+global repository `c(pwd)' //returns current working directory
+
+***********************
+//set operating system*
+***********************
+global os `c(os)'
+
+***********************************************
+//Run all code in all folders in correct order*
+***********************************************
+
+//Run code in main analysis
+cd "$repository/main/"
+foreach file in main analysis{
+	do "`file".do
+}
+
+//Run code in reviewers analysis
+cd "$repository/reviewers/"
+foreach file in reviewers analysis{
+	do "`file".do
+}
+
+//Run code in robustness analysis
+cd "$repository/robustness/"
+foreach file in robustness analysis{
```

Comment:
```suggestion
foreach file in robustness_property_chars robustness_state robustness_host edelman_price {
```

Why not the above? aren't we running all the files in "robustness folder"? but there's no file named "robustness.do" and no file named "analysis.do", so i'm not sure what this will be running...